accel_code/bids_transform.py

```diff
@@ -14,9 +15,6 @@ def bids_transform(target_project, sub_id, sess_id):
 
     data_path = utils.get_test_data_path(target_project)
 
-    # Initialize the layout
-    layout = BIDSLayout(data_path)
-
     entities = {
     'subject': sub_id,
```

Comment:
depending on the target project, we may need an `if statement` to modify the `sub_id`. For example, in the BETTER project, the ids look like: `sub-GE120001` (GE indicates the scanner), so all `sub_ids` need a `GE` pre-pended to them. `Bike_Atrain` also has special prefixes, but that may require a more general solution for finding the `sub_id` in the directory and preprending the arbitrary string associated with the directory name.

For example, if I have `sub_id = 326` and in `Bike_Atrain` the directory is `sub-GEP326`, then I would have to search through the `sub-AAAXXX` directory names in Bike_Atrain (think [glob](https://docs.python.org/3/library/glob.html)) to see if 326 is in the directory string name, and once I find the directory, prepend or use the sub_id found in the target directory.

I explained this somewhat unclearly, please let me know if you have questions about this.